management-server/internal/grpc-server/server.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"time"
+
+	apiProtos "github.com/wso2/apk/management-server/internal/grpc/api"
+	logger "github.com/wso2/apk/management-server/internal/loggers"
+	"github.com/wso2/apk/adapter/pkg/logging"
+	"github.com/wso2/apk/management-server/pkg/tlsutils"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/keepalive"
+	"github.com/wso2/apk/management-server/config"
+)
+
+type aPIService struct {
+	apiProtos.UnimplementedAPIServiceServer
+}
+
+func NewApiService() *aPIService {
+	return &aPIService{}
+}
+
+func (s *aPIService) CreateAPI(ctx context.Context, api *apiProtos.API ) (*apiProtos.Response, error) {
+	logger.LoggerMGTServer.Printf("Message received : %q", api);
```

Comment:
please use debug/info/error when adding logs. Also adhere to the proper logging structure. you can refer, https://docs.google.com/document/d/1sLOvjK760QXU09r3RdrI_I_v-3L1lX2VUQKv0gN1ndY/edit#heading=h.q1urrm0go3p and update the error codes in https://docs.google.com/spreadsheets/d/1VH2rMZ-PdPbFw6VjUWV0o1Y0gxKANuVhgUpdYD1Iiog/edit#gid=0